From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jayesh Choudhary <j-choudhary@ti.com>
Date: Fri, 11 Nov 2022 16:20:03 +0530
Subject: [PATCH] soc: ti: k3: ringacc: drop loglevel for non-fatal probe
 deferral log

This error is expected due to probe deferral. It is non-fatal and
it will probe again later on. So drop the loglevel from dev_err()
to dev_dbg().

Fixes: 3277e8aa2504 ("soc: ti: k3: add navss ringacc driver")
Signed-off-by: Jayesh Choudhary <j-choudhary@ti.com>
---
 drivers/soc/ti/k3-ringacc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/ti/k3-ringacc.c b/drivers/soc/ti/k3-ringacc.c
index 086d61a86b17..16500bfe4a08 100644
--- a/drivers/soc/ti/k3-ringacc.c
+++ b/drivers/soc/ti/k3-ringacc.c
@@ -1358,7 +1358,7 @@ static int k3_ringacc_init(struct platform_device *pdev,
 	dev->msi_domain = of_msi_get_domain(dev, dev->of_node,
 					    DOMAIN_BUS_TI_SCI_INTA_MSI);
 	if (!dev->msi_domain) {
-		dev_err(dev, "Failed to get MSI domain\n");
+		dev_dbg(dev, "Failed to get MSI domain\n");
 		return -EPROBE_DEFER;
 	}
 
